eurostag-model/src/main/java/com/powsybl/eurostag/model/EsgGenerator.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.eurostag.model;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class EsgGenerator {
+
+    private final Esg8charName znamge; // generator name
+    private final EsgConnectionStatus xgenest; // status
+                                            // ‘Y‘     : connected
+                                            // ‘N’     : not connected
+    private final Esg8charName znodge; // connection node name
+    private final double pgmin; // minimum active power [MW]
+    private final double pgen; // active power [MW]
+    private final double pgmax; // maximum active power [MW]
+    private final double qgmin; // minimum reactive power [Mvar]
+    private final double qgen; // reactive power [Mvar]
+    private final double qgmax; // maximum reactive power [Mvar]
+    private EsgRegulatingMode xregge; // regulating mode
+    private double vregge; // voltage target
+    private final Esg8charName zregnoge; // regulated node (= ZNODGE if blank)
+    private final double qgensh; // Reactive sharing coefficient [%]
```

Comment:
I think that we have to use the extension `CoordinatedReactiveControl` to store the reactive sharing coefficient. It seems that we don't do that in the convserion.

Reply:
When the first version of this exporter has been implemented, the extension you are talking about didn't exist. I agree with your proposal: do you think we should do it in this PR or create an issue and do it later?

Reply:
I will find time to make it in that PR.